pkg/trace/api/profiles.go

```diff
@@ -22,48 +25,74 @@ const (
 	profilingURLDefault = "https://intake.profile.datadoghq.com/v1/input"
 )
 
-// profilingEndpoint returns the profiling intake API URL based on agent configuration.
-func profilingEndpoint() string {
+// profilingEndpoints returns the profiling intake urls and their corresponding
+// api keys based on agent configuration. The main endpoint is always returned as
+// the first element in the slice.
+func profilingEndpoints(apiKey string) []*traceconfig.Endpoint {
+	var endpoints []*traceconfig.Endpoint
+	main := profilingURLDefault
 	if v := config.Datadog.GetString("apm_config.profiling_dd_url"); v != "" {
```

Comment:
I think we might want all of these settings (URL and additional endpoints) to be settable via environment variables, as that's the common way in containerized environments. It should be more or less a copy paste of what we already have for `apm_config.apm_dd_url` and `apm_config.additional_endpoints` in `pkg/trace/config/env.go`.

We'll probably need:
* `DD_APM_PROFILING_ADDITIONAL_ENDPOINTS`
* `DD_APM_PROFILING_DD_URL`